packages/core/src/exchanges/dedup.ts

```diff
@@ -3,6 +3,7 @@ import { Exchange, Operation, OperationResult } from '../types';
 
 /** A default exchange for debouncing GraphQL requests. */
 export const dedupExchange: Exchange = ({ forward, client }) => {
+  const dispatchEvent = client.debugTarget!.dispatchEvent;
```

Comment:
This won't be defined (read: will throw) and will bypass the babel plugin, let's keep doing this in full (or edit the babel plugin to remove this as well)